scss/skin/_default.scss

```diff
@@ -67,6 +67,11 @@ video {
   @extend .oo-expand;
 }
 
+// fix anamorphic video issue on Firefox
+@include apply-to-firefox-only ('.oo-anamorphic video') {
+  object-fit: fill !important;
```

Comment:
Will this work in fullscreen as well, or will it distort the aspect ratio by filling up the viewport?